R/ml_kmeans.R

```diff
@@ -54,19 +56,33 @@ ml_kmeans <- function(x,
   # extract cluster centers
   kmmCenters <- invoke(fit, "clusterCenters")
 
+  # compute cost for k-means
+  if (compute.cost)
+    kmmCost <- invoke(fit, "computeCost", tdf)
+
   centersList <- transpose_list(lapply(kmmCenters, function(center) {
     as.numeric(invoke(center, "toArray"))
   }))
 
   names(centersList) <- features
   centers <- as.data.frame(centersList, stringsAsFactors = FALSE, optional = TRUE)
 
-  ml_model("kmeans", fit,
-           centers = centers,
-           features = features,
-           data = df,
-           model.parameters = as.list(envir)
-  )
+  if (compute.cost) {
```

Comment:
I think it would be okay to have a single call to `ml_model`; just have the default `cost` be `NULL`.